src/components/ButtonPanel.js

```diff
@@ -1,7 +1,50 @@
-export default function ButtonPanel() {
-  return (
-    <h1>
-      Button Panel
-    </h1>
-  );
-}
+import React from 'react';
+import Button from './buttons/button';
+
+/* eslint-disable */
+
```

Comment:
- [ ] It would be better if we group the elements in the following order:
![Screenshot from 2021-09-16 14-04-02](https://user-images.githubusercontent.com/46388113/133626820-dd8f9b82-8514-437a-b2ed-ee3e6a81a382.png)

Reply:
![image](https://user-images.githubusercontent.com/77872355/133629212-b64e2901-8f3e-41c3-94b4-22428a7f2248.png)
It is that like order on the screen, I have grouped them on arrays to place them on that like groups